src/commands/watch.js

```diff
@@ -19,8 +19,7 @@ export default asyncCommand({
 		},
 		host: {
 			description: 'Hostname to start a server on',
-			default: '0.0.0.0',
-			alias: 'h'
```

Comment:
removing the alias for the `host` command since this is colliding with `yargs` package's `--help, -h`.